ODPi-Runtime.md

```diff
@@ -273,7 +473,7 @@ Best practices for compatible apps to be portable and operator friendly:
 
 -   Applications SHOULD NOT assume that an Oracle JRE is being used.
 
--   Applciations SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
+-   Applications SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
```

Comment:
The changes above are created by the IBM team to make sure the variables and environment are clear and available for HCFS.